src/components/item/sharing/ItemSharingTab.tsx

```diff
@@ -85,7 +88,7 @@ const ItemSharingTab = ({ item }: Props): JSX.Element => {
         todo: show only if item is pseudomized
         */}
         {/* // todo: this will change with the refactor */}
-        {getItemLoginSchema(item?.extra as any) && (
+        {getItemLoginSchema(item?.extra as { itemLogin?: ItemLogin }) && (
```

Comment:
Why is the cast still necessary here?

Reply:
I am not sur how the `itemLogin` extra works. We may have to adjust the type to make this cast caduc, for the moment I don't think there is a way of removing it.

Reply:
I am not sur how the `itemLogin` extra works. We may have to adjust the type to make this cast caduc, for the moment I don't think there is a way of removing it.

Reply:
@pyphilia can you help us out here ? Is the `itemLogin` extra only available on folders ?